mpv-wrapper

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+ln -snf "$(xdg-user-dir PICTURES)" "$HOME/.var/app/io.mpv.Mpv/Pictures"
```

Comment:
Maybe check if the symlink exists first rather than creating symlink all the time

Reply:
I don't think that would be a good idea. What if the user ends up changing their `user-dirs.dirs`? The directory would be pointing at the old location.